pkg/volume/plugins.go

```diff
@@ -164,6 +164,11 @@ type VolumeHost interface {
 	// implemented in terms of the "emptyDir" volume.
 	NewWrapperMounter(volName string, spec Spec, pod *api.Pod, opts VolumeOptions) (Mounter, error)
 
+	// NewWrapperAttacher creats an Attacher which wraps one belonging to
+	// another volume. This is used to create Attachers for plugins which
+	// wrap other plugins such as the persistentClaimPlugin
+	NewWrapperAttacher(volName string, spec Spec, pod *api.Pod) (Attacher, error)
```

Comment:
This is really unnecessary. 

Consider how this would work if we were to go in this direction: the plugins would have to dereference the PVC/PV to get at the physical volume, which would require the plugins to make calls to the API server to fetch those objects.

On the attach/detach controller side we dereference the PVC/PV where it is encountered and then operate strictly on VolumeSpec.

There is no need to push those dependencies down into the plugin. So I think the existing interface is fine.